### ISSUE TITLE
[Feature Request]: Filename for code generation

### ISSUE THREAD

**Issue by piperone** [laravel-idea/plugin#982]
### Feature Description

This is more of a question than a feature request, really. I have created a custom code generation that uses a custom file template. It's a variation of the db migration code generation. But it appears that I can't control the filename, which by convention is timestamp + filename snakecased. Is there some way I can achieve this for my custom generation?

**Comment by adelf** [comment]
Yes. Please check https://github.com/TheDragonCode/laravel-actions/blob/main/ide.json

**Comment by piperone** [comment]
I don't have an ide.json-file in my project. I am able to find my custom generations inside the .idea-folder in a file called laravel-idea.xml. I tried adapting the thing from your snippet and put it in the xml-file:

```xml
<CustomGeneration>
    <option name="directory" value="database/migrations" />
    <option name="name" value="${CURRENT_TIME|format:yyyy_MM_dd_HHmmss}_${INPUT_CLASS|className|snakeCase}.php" />
    <option name="parameters">
```

... but that did not work.

**Comment by adelf** [comment]
Sorry, I thought you were talking about ide.json generations... It's not possible for custom generations. I'll try to solve this somehow.
